src/main/java/com/example/demo/services/LeaseManagementDropBoxImpl.java

```diff
@@ -100,4 +112,32 @@ public void dropboxCallback(Long leaseId) throws ApiException, EmptyResultDataAc
         //todo check resonse object
         log.info("lease status updated for leaseId: {}", leaseId);
     }
+
+    ZonedDateTime parseZonedDateTime(String dateStr) {
+        return LocalDate.parse(dateStr).atStartOfDay(ZoneId.of("UTC"));
+    }
+
```

Comment:
You could also work with ZonedDateTime objects in the DTO. Then the frontend is responsible to give the exact moment in time for the start date and end date. Or you should get the information about the timezone the property is in.